app/src/main/java/org/fossasia/susi/ai/chat/ChatPresenter.kt

```diff
@@ -270,12 +272,14 @@ class ChatPresenter(chatActivity: ChatActivity) : IChatPresenter, IChatModel.OnR
     override fun onLocationSuccess(response: Response<LocationResponse>) {
         if (response.isSuccessful && response.body() != null) {
```

Comment:
All null checks below are because `request.body()` is null asserted everywhere. Just assign response.body() to a new variable above the block and check nullability once and it will be fine in the body